docs/assets/less/dialtone-docs.less

```
@@ -273,15 +274,18 @@
 }
 
 .hero--inner-wrapper {
-  padding-top: 12.8rem;
   padding-bottom: 12.8rem;
+  padding-top: 12.8rem;
   @media (max-width: 640px) {
-    flex-direction: column;
+    flex-direction: column
```

```suggestion
    flex-direction: column;
```

Looks like we accidentally deleted a semi colon here

---

👀 👍